tasks/src/robocorp/tasks/__init__.py

```diff
@@ -162,6 +165,25 @@ def get_current_task() -> Optional[ITask]:
     return _task.get_current_task()
 
 
+def inject_truststore():
+    # Use certificates from native storage (if `truststore` installed)
+    if os.environ.get("RC_CHECK_TRUSTSTORE_DEP", "True") in ["false", "False", 0]:
```

Comment:
🐛 
```suggestion
    if os.getenv("RC_CHECK_TRUSTSTORE_DEP", "True").lower() in ["false", "0"]:
```
Note that you'd always get a string from this one, so if you pass `RC_CHECK_TRUSTSTORE_DEP=0` you'd end up with `"0"`.

Also, an `os.getenv()` might look better here + lower case normalization, but your call.

----

About on the `RC_CHECK_TRUSTSTORE_DEP`, not sure if that's the right name, as this isn't just a check, but it also affects behaviour (when doing the injection), so I would rename it into `RC_USE_TRUSTSTORE` and then I would document somewhere in the docs on how the automatic injection can be disabled.